semantic_search/main.py

```diff
@@ -106,7 +106,7 @@ async def query(query: Query) -> List[Dict[str, float]]:
         add_to_faiss_index(ids, embeddings, model.index)
 
     # Can't search for more items than exist in the index
-    top_k = min(model.index.ntotal, query.top_k)
+    top_k = min(model.index.ntotal, query.top_k) + 1
```

Comment:
I think it should be: `top_k = min(model.index.ntotal, query.top_k + 1)`. Doesn't make sense to request more items than are in the index, right?

Reply:
oh yeah, that was my fault. I should've seen that. But yeah that fixes it.